no-computer.md

```diff
@@ -11,34 +11,65 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+The need is for people who want to restrict access to something. Before locks
+were invented, people would have to either trust that people won't access the
+thing they want to protect, hire someone to watch over it, or hide it. Since
+locks, people can be secure in that others will not enter where they should not
+go or steal their belongings.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Users want to be able to enter a short program that only they know (a
+combination) that will cause the language to grant access, and deny access to
+everyone else. People can easily restrict access to the knowledge of this
+program so that only the ones they trust know it, and the language of a lock
+keeps everyone else out.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+Locks are cool because each one is like its own language. Combinations and keys
+are programs, and these programs have similar structure, but each lock accepts
+a different program. There can also be some languages that accept multiple
+programs, such as a "standard" and "master" key.
```

Comment:
How interesting. You've just connecting designing lock types and designing languages. Cool